marc_to_solr/lib/authority_traject_config.rb

```diff
@@ -14,15 +14,18 @@
   provide "solr_writer.max_skipped", "50"
   provide "marc4j_reader.source_encoding", "UTF-8"
   provide "log.error_file", "./log/traject-error.log"
-  provide "allow_duplicate_values",  false
+  provide "allow_duplicate_values", false
   provide "solr_writer.commit_on_close", "true"
 end
 
 $LOAD_PATH.unshift(File.expand_path('../../', __FILE__)) # include marc_to_solr directory so local translation_maps can be loaded
 
-to_field 'id', extract_marc('001', first: true)
+to_field 'auth_001_s' do |record, accumulator|
+  value = record['001'].gsub(/\s+/, '') if recorc['001']
```

Comment:
record instead of recorc. This should pull from the 010 field instead of the 001 field

Reply:
right

Reply:
I went again through my notes and we said we will index the 001 with no spaces.

Reply:
oh ok

Reply:
so we do that already :)

Reply:
I put back the id that pulls from the 001. We keep the new field because @mzelesky  said that it needs to be with no spaces.